### PR TITLE
Updated data.json

### PR DESCRIPTION
**Pull request by JayantGoel001**
Resolve #

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -994,5 +994,11 @@
...
     "imageUrl": "https://user-images.githubusercontent.com/10162710/120894576-6ff8f200-c619-11eb-9ee5-5078912d5550.png",
     "issueNumber": 552
-  }
+  },
+  {
+    "name": "Kamlesh Joshi",
+    "githubUsername": "kamleshjoshi8102",
+    "imageUrl": "https://user-images.githubusercontent.com/72374645/123034023-67b8f900-d406-11eb-829a-6df34ba91852.png",
+    "issueNumber": 558
+   }
```
Comment (schmelto) [inline data.json:1003]:
```suggestion
  }
```
Think there is a space too much

File: data.json [hunk 2]
```diff
@@ -994,5 +994,11 @@
     "githubUsername": "bevennyamande",
     "imageUrl": "https://user-images.githubusercontent.com/10162710/120894576-6ff8f200-c619-11eb-9ee5-5078912d5550.png",
     "issueNumber": 552
-  }
+  },
+  {
+    "name": "Kamlesh Joshi",
+    "githubUsername": "kamleshjoshi8102",
+    "imageUrl": "https://user-images.githubusercontent.com/72374645/123034023-67b8f900-d406-11eb-829a-6df34ba91852.png",
+    "issueNumber": 558
```
Comment (schmelto) [inline data.json:1002]: 558 seems to me to be the wrong issue number

File: data.json [hunk 3]
```diff
@@ -994,5 +994,11 @@
     "githubUsername": "bevennyamande",
     "imageUrl": "https://user-images.githubusercontent.com/10162710/120894576-6ff8f200-c619-11eb-9ee5-5078912d5550.png",
     "issueNumber": 552
-  }
+  },
+  {
+    "name": "Kamlesh Joshi",
+    "githubUsername": "kamleshjoshi8102",
```
Comment (schmelto) [inline data.json:1000]: Why do you want to add someone else?

Comment (JayantGoel001) [reply data.json:1000]: Actually, That person committed the changes to my forked Repo😅.
So I thought to create a PR for it.

Comment (JayantGoel001) [reply data.json:1000]: Can I do it?
Or Should I ask him to create a PR on this Repo?

Comment (schmelto) [reply data.json:1000]: > Can I do it?
> Or Should I ask him to create a PR on this Repo?

Tough question I think its OK when you open a pr for this.

**Review by schmelto — APPROVED** [review]
Lgtm

**Review by vinzvinci — APPROVED** [review]
LGTM 👍

**Review by schmelto — APPROVED** [review]

**Pull request merged**